src/dojos/cd-find-dojo.vue

```diff
@@ -301,18 +355,49 @@
               flex: none;
               margin-right: 0;
               min-width: auto;
+
+              &--reduced {
+                min-width: 70%;
+                flex: none;
+                margin-right: 4px;
+              }
             }
             &-submit {
               margin-top: 8px;
               > .btn {
                 width: 100%;
               }
+              &--reduced {
+                margin-top: none;
+                flex: 1;
+                > .btn {
+                  background: #2a8244;
+                  min-width: 60px;
```

Comment:
min-width ? Isn't that some kindof quirk that won't work with another language ?